container_api/src/container.rs

```diff
@@ -397,14 +413,14 @@ pub mod tests {
     handle = "DPKI"
 
     [[bridges]]
-    caller_id = "test-instance-3"
+    caller_id = "bridge-caller"
     callee_id = "test-instance-2"
     handle = "happ-store"
 
     [[bridges]]
-    caller_id = "test-instance-3"
+    caller_id = "bridge-caller"
     callee_id = "test-instance-1"
-    handle = "DPKI"
+    handle = "test-callee"
```

Comment:
This handle is the "instance_handle" that the bridge caller has to provide to `hdk::call` to reference this bridged instance.